examples/wafv2/webaclassociation.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: wafv2.aws.upbound.io/v1beta1
+kind: WebACLAssociation
+metadata:
+  annotations:
+    meta.upbound.io/example-id: wafv2/v1beta1/webaclassociation
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    region: us-west-1
+    resourceArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    webAclArnSelector:
```

Comment:
Unless we also configure `wafv2.WebACL` and reference it here, this example manifest will not be testable on its own. If you decide not to configure `wafv2.WebACL` within this PR, then we will ask you to manually test `wafv2. WebACLAssociation` and report the results in the PR description and add the `upjet.upbound.io/manual-intervention` [with its reasoning](https://github.com/upbound/provider-aws/blob/ecb2053d23a951507c8ea03ea195d7d9db5c5420/examples/ec2/tag.yaml#L5). Or, if you decide to also configure the `wafv2.WebACL`, then we can add it as a dependency in this manifest and [uptest this resource](https://github.com/upbound/uptest). We will help you in uptesting this resource during the reviews if you choose to follow that path.